linalg/blockoperator.hpp

```diff
@@ -74,7 +74,7 @@ class BlockOperator : public Operator
    //! Check if block (i,j) is a zero block
    int IsZeroBlock(int i, int j) const { return (op(i,j)==NULL) ? 1 : 0; }
    //! Return a reference to block i,j
-   Operator & GetBlock(int i, int j)
+   Operator & GetBlock(int i, int j) const
```

Comment:
This `const` does not seem to be necessary for building the library, examples, and miniapps. Is it really needed?

Reply:
I think for some reason I had to add it when I hacked in the direct system solve. I'll remove it.